src/dali_model_instance.h

```diff
@@ -55,11 +55,16 @@ class DaliModelInstance : public ::triton::backend::BackendModelInstance {
 
   void Execute(const std::vector<TritonRequest>& requests);
 
+  void ExecuteBatched(const std::vector<TritonRequest>& requests);
+
+  void ExecuteUnbatched(const std::vector<TritonRequest> &requests);
```

Comment:
I see that `Execute` handles both `ExecuteBatched` and `ExecuteUnbatched`. Do these functions have to be public?

Reply:
done